internal/controller/linstorcluster_controller.go

```diff
@@ -453,6 +468,14 @@ func (r *LinstorClusterReconciler) kustomizeCSIControllerResources(lcluster *pir
 		patches = append(patches, p...)
 	}
 
+	t := tolerations.MergeTolerations(tolerations.LinstorControllerTolerations, tolerations.HAControllerTolerations, lcluster.Spec.Tolerations)
```

Comment:
Why does `linstor-csi-controller` get the `LinstorControllerTolerations`? The comment on `FailOverTimeOut` indicates that it applies only to the LINSTOR Controller.